modules/assim.batch/R/autoburnin.R

```diff
@@ -0,0 +1,62 @@
+#' @name getBurnin
+#' @title Calculate burnin value
+#' @description Use `coda::gelman_plot` to calculate first iteration number where Gelman diagnostic never exceeds the specified threshold.
+#' @param jags_out List of MCMC sample matrices or `mcmc.list` object
+#' @param threshold Maximum value of Gelman diagnostic
+#' @param ... Other parameters to `gelman.plot`
+#' @examples
+#'      library(coda)
+#'      data(line)
+#'      burnin <- getBurnin(line, threshold = 1.05)
+#' @author Michael Dietze, Alexey Shiklomanov
+#' @export
+getBurnin <- function(jags_out, threshold = 1.1, ...) {
+    library(coda)
+    if (length(find("logger.info")) == 0) {
+        msg <- message
+    } else {
+        msg <- logger.info
+    }
+    if (!is.mcmc.list(jags_out)) jags_out <- makeMCMCList(jags_out)
```

Comment:
Another good catch! It's currently in `PEcAnRTM`, but migrated it to `assim.batch` in the upcoming commit.